Gemfile.lock

```diff
@@ -180,6 +181,7 @@ GEM
     version_gem (1.1.4)
 
 PLATFORMS
+  arm64-darwin-22
```

Comment:
It's weird that [the error](https://github.com/umts/strap/actions/runs/11977229119/job/33494615742#step:13:75) says `your local platform is arm64-darwin-22`, but the [Image Release](https://github.com/actions/runner-images/releases/tag/macos-14-arm64%2F20241119.509) says it's `Kernel Version: Darwin 23.6.0`.

https://github.com/umts/strap/actions/runs/11977229119/job/33494615742#step:1:12

Reply:
Maybe add a `bundle platform` step and see if `arm64-darwin-23` is supported?